provisioning/roles/container.osx/tasks/xhyve.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Thanks to https://allysonjulian.com/setting-up-docker-with-xhyve/
+- name: Install xhyve and dependencies
+  homebrew:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - xhyve
+    - docker
+    - docker-compose
+    - docker-machine
+
+- name: Download hyve driver for docker
+  get_url:
+    url: https://github.com/zchee/docker-machine-driver-xhyve/releases/download/v0.2.2/docker-machine-driver-xhyve
+    dest: /usr/local/bin/docker-machine-driver-xhyve
+    checksum: "sha1:9af9650a697336f1b00ef64475d7ee952d66b9f3"
+    # BEWARE: get_url + file is not atomic. So if the following lines are
+    # uncommented, and the 'file' part of the task fails, the whole
+    # task fails but on next run it will succeed because the precondition
+    # dest+checksum is already satisfied. Thus, we need to put the
+    # file module on a separate task
+    # owner: root
+    # group: wheel
+    # mode: ug+x,u+s
```

Comment:
Why? It makes sense, as it seems logical to merge the two commands into one, to tell any future developer the exact reason we don't use owner/group/mode in get_url command.

Maybe just link to the issue?